build.sbt

```diff
@@ -6,7 +6,7 @@ organization in Global := "edu.gemini.ocs"
 
 ocsVersion in ThisBuild := OcsVersion("2016A", true, 1, 1, 1)
 
-pitVersion in ThisBuild := OcsVersion("2016A", false, 1, 1, 0)
+pitVersion in ThisBuild := OcsVersion("2016B", true, 2, 1, 0)
```

Comment:
Your explanation makes sense, but it does seem strange to start with version 2...